examples/03_special_classes/04_Object.md

```diff
@@ -1,21 +1,20 @@
-# Object keyword
+# Object Keyword
 
-First of all, let's start with some basic OOP concepts: a *class* is a blueprint, and an *object* is an instance of a class.
-You define a class, then create multiple instances of that class:
+Classes and objects in Kotlinwork the same way as in most object-oriented languages: a *class* is a blueprint, and an *object* is an instance of a class. Usually, you define a class and then create multiple instances of that class:
```

Comment:
Classes and objects in Kotlin work the same way as in most object-oriented languages: a *class* is a blueprint, and an *object* is an instance of a class. Usually, you define a class and then create multiple instances of that class:

just a space between Kotlin and work is needed.